Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -225,7 +225,12 @@ def print_caveats
   def enable_accessibility_access
     return unless @cask.accessibility_access
     ohai "Enabling accessibility access"
-    if MacOS.version <= :mountain_lion
+    if MacOS.version >= :sierra
```

Comment:
We generally favour using `<=` and `else` while “climbing”. So always say `if <= oldest … elsif <= old … elsif <= less_old … else`. So we climb up versioning and leave `else` as something that means “from this version on”.